libmscore/cmd.cpp

```diff
@@ -2443,6 +2443,55 @@ Element* Score::move(const QString& cmd)
             auto measureBase = cr ? cr->measure()->findMeasureBase() : box->findMeasureBase();
             el = measureBase ? cmdNextPrevFrame(measureBase, false) : nullptr;
             }
+      else if (cmd == "next-section") {
```

Comment:
These haven't been tested, will download artifact tomorrow and try it out, and if it all works fine, could easily fix this up to be more modular looking. Something's up with "make revision" in 3.x for me currently.